Updating plan names. (#1298)
RedisLabs/redislabs-docs: content/rc/security/database-security/tls-ssl.md

aliases: /rc/administration/security/securing-redis-cloud-connections
---

Transport Layer Security (TLS), commonly called "SSL", ensures the privacy of data sent between applications and their Redis databases. TLS is available in Redis Cloud Flexible and Annual subscriptions.

> \

Reviewer: @lanceleonard Is this "\" intentional?

Author: Not by me.  :-)

(I hadn't done a copy edit on the content.  Was trying to only update the plan names.  The more recent commits include a quick copy edit and updated images for the certificate authorization buttons.)

5. To require client authentication, select **Enforce client authentication**.

    ![Enforce Client Authentication](/images/rc/enforce-client-auth.png "SSL Client Authentication")

6. Next, either provide an X.509 certificate containing a public key for your client or use the 
**Generate Client Certificate** button to generate one.

7. If you generate your certificate using the admin console, then a download will begin containing the following artifacts:
   * `redislabs_user.crt` – the certificate's public key.
   * `redislabs_user_private.key` – the certificate's private key.
   * `redislabs_ca.pem` – the Redis Cloud CA certificate.

8. Select the **Update** button to apply your changes and enable TLS.

    ![Cancel / Update](/images/rc/cancel-update.png "Cancel / Update")

{{<note>}}
Once you've enabled TLS, all client connections to your database must use TLS. Unencrypted connections
will no longer be permitted.
{{</note>}}

## Connect over TLS

To connect to a Redis Cloud database over TLS, you will need:

* A Redis client that supports TLS

> * The Redis Cloud client authentication (CA) certificate

Reviewer: Here, CA looks like an abbreviation for "client authentication", but it's not. I'd remove the parentheses.

Author: Yes; that's intentional.

> * The Redis Cloud client authentication (CA) certificate

Reviewer: ```suggestion
* The Redis Cloud CA certificate
```

Author: Fixed.

### Certificates

If you don't have the Redis Cloud CA certificate, you can download it from the admin
console by going to **Settings** and selecting the **Flexible Plans CA** button.

![Flexible Plans CA button](/images/rc/rc-settings-ca-flexible.png "Flexible Plans CA")

> If you're requiring client authentication, you'll also need a public and private client keys. See

Reviewer: ```suggestion
If you're requiring client authentication, you'll also need public and private client keys. See
```

Author: Fixed.

[Enabling TLS](#enabling-tls) for detai.

### Connect with the Redis CLI

> Here's how to use the Redis command-line interface (CLI) tool to a TLS-enabled Redis Cloud database:

Reviewer: ```suggestion
Here's how to use the Redis CLI to connect to a TLS-enabled Redis Cloud database:
```

Author: Fixed.
